cg/meta/orders/microbial_fastq_submitter.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+
+from cg.constants import Workflow, DataDelivery
+from cg.exc import OrderError
+from cg.meta.orders.lims import process_lims
+from cg.meta.orders.submitter import Submitter
+from cg.models.orders.order import OrderIn
+from cg.models.orders.sample_base import StatusEnum
+from cg.store.exc import EntryNotFoundError
+from cg.store.models import Sample, Customer, Case, CaseSample
+from cg.utils.error_handler import handle_errors
+
+
+class MicrobialFastqSubmitter(Submitter):
+    @handle_errors(to_except=(EntryNotFoundError,), to_raise=OrderError)
+    def submit_order(self, order: OrderIn) -> dict:
+        project_data, lims_map = process_lims(
+            lims_api=self.lims, lims_order=order, new_samples=order.samples
+        )
+        status_data: dict = self.order_to_status(order)
+        self._fill_in_sample_ids(samples=status_data["samples"], lims_map=lims_map)
+        new_samples: list[Sample] = self.store_items_in_status(
+            customer_id=status_data["customer"],
+            order=status_data["order"],
+            ordered=project_data["date"] if project_data else datetime.now(),
+            ticket_id=order.ticket,
+            items=status_data["samples"],
+        )
+        self._add_missing_reads(new_samples)
```

Comment:
I guess the `_add_missing_reads` should be removed? Or did we not have a PR open for removing it?